README.md

```diff
@@ -13,24 +13,24 @@ mdns adds multicast DNS service discovery, also known as zeroconf or bonjour to
 
 ```js
 // import the module
-var mdns = require('mdns');
+import mdns from 'mdns';
```

Comment:
If only this worked out of the box :)

Reply:
Got it. I'll revert the import statements